buildpipeline/pipeline.json

```diff
@@ -121,6 +124,7 @@
           "Parameters": {
             "PB_TargetArchitecture": "x64",
             "PB_PortableBuild": "true",
+            "PB_AdditionalMSBuildArguments": "/p:SkipTests=$(PB_SkipTests) /p:DotNetRestoreSources=$(PB_RestoreSource)",
```

Comment:
Why do we sometimes `-SkipTests` and other times `/p:SkipTests`?

Reply:
I did't rework all the definitions so if they were parameters to build.sh/cmd then it is just `-SkipTests` but if they were msbuild args then just `/p:SkipTests`. While I don't like the inconsistency either I didn't want to rework the entire build defintion at this point in time.

Reply:
fair enough.  it would be nice to be consistent.